### ISSUE TITLE
Support for bytea casting?

### ISSUE THREAD

**Issue by gjtorikian** [pksunkara/pgx_ulid#42]
Hi, I noticed there were functions to cast to/from uuid. Would you accept a PR to do so from bytea?

**Comment by gjtorikian** [comment]
Eh, I suppose `_send` works:

```
SELECT uuid_send(gen_ulid()::uuid);
             uuid_send              
------------------------------------
 \x018f91e53b9e69708a4b26bdc784d0e3
(1 row)

SELECT pg_typeof(uuid_send(gen_ulid()::uuid));
 pg_typeof 
-----------
 bytea
(1 row)
```

**Comment by pksunkara** [comment]
I would welcome a PR for bytea casting.

**Comment by pksunkara** [comment]
Released 0.1.5
